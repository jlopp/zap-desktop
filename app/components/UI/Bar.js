import React from 'react'
import { Box } from 'rebass'

/**
 * @render react
 * @name Bar
 * @example
 * <Bar />
 */
class Bar extends React.PureComponent {
  render() {
    return <Box width={1} bg="white" css={{ height: '1px' }} {...this.props} as="hr" />
  }
}

export default Bar
