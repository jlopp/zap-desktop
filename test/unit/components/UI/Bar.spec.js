import React from 'react'
import Bar from 'components/UI/Bar'
import renderer from 'react-test-renderer'

describe('component.UI.Bar', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Bar />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
