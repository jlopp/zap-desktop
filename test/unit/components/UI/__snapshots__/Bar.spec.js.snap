// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.Bar should render correctly 1`] = `
.c0 {
  width: 100%;
  background-color: white;
  height: 1px;
}

<hr
  className="c0"
  width={1}
/>
`;
